final_task/rss_reader/args_creater.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+
+
+def arguments():
+    """create command line arguments"""
+
+    parser = argparse.ArgumentParser(description='Pure Python command-line RSS reader')
+    # source url
```

Comment:
Here and below: there is no need to be that verbose in your comments. Usually comments are left when they are followed by a difficult to read or difficult to understand piece of code.